client/client-multi/client-c/src/kaa/platform-impl/common/key_utils.c

```diff
@@ -140,7 +140,7 @@ kaa_error_t ext_get_sha1_base64_public(const uint8_t **sha1, size_t *length)
         return KAA_ERR_BADPARAM;
     }
     static size_t sha1_base64_len = 0;
-    static uint8_t sha1_base64_buffer[KAA_SHA1_PUB_BASE64_LEN];
+    static uint8_t sha1_base64_buffer[KAA_SHA1_PUB_BASE64_LEN + 1];
```

Comment:
I don't think so, since the actual length of base64-encoded data is 28, but the `sha1_base64_buffer` will contain the string representation (additional byte for NULL terminator).